_rules/iframe-accessible-name-cae760.md

```diff
@@ -21,23 +21,23 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to `iframe` elements that are [included in the accessibility tree][] or that are part of [sequential focus navigation][].
+The rule applies to `iframe` elements that are [included in the accessibility tree][] and has contents that can be accessed by [sequential focus navigation][].
```

Comment:
This one keeps getting more complicated... :( I think we made a mistake with the "is in sequential focus navigation". I did the following tests with Voiceover:

| scenario  | VO result |
|---------|------------|
| empty frame | VO ignores it completely
| empty iframe with tabindex=1 | VO ignores it completely
| iframe with focusable content | VO announces the iframe
| iframe with non-focusable content | VO announces the iframe

Interesting caveat; VO in Safari does not look at aria-label. It only announces title. We should put that in Accessibility support.


This tells me what matters here, at least for VO is if the iframe has any content. I tested with an image with alt="", if that's the only thing in the frame, VO ignores it. I think we need to do more testing before we continue with this one.

Reply:
That's... concerning 😮